src/App.jsx

```diff
@@ -1,55 +1,33 @@
-import React, { useState, useEffect } from "react";
-import { BrowserRouter as Router } from "react-router-dom";
-import "./App.css";
-import Login from "./components/Login";
-import Register from "./components/Register";
-
-const App = () => {
-  const [loading, setLoading] = useState(true);
-  const [showLogin, setShowLogin] = useState(false);
-  const [showRegister, setShowRegister] = useState(false);
-
-  // Loading delay if we want to add animation or logo later
-
-  useEffect(() => {
-    const timer = setTimeout(() => {
-      setLoading(false);
-    }, 2000);
-
-    return () => clearTimeout(timer);
-  }, []);
-
-  const handleLoginClick = () => {
-    setShowLogin(true);
-    setShowRegister(false);
-  };
-
-  const handleRegisterClick = () => {
-    setShowRegister(true);
-    setShowLogin(false);
-  };
-
+import React from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Navbar from "./components/Navigation/Navbar";
+import About from "./pages/About";
+import Contact from "./pages/Contact";
+import SignIn from "./pages/SignIn";
+import Register from "./pages/Register";
+import Home from "./pages/Home";
+import Dashboard from "./pages/Dashboard";
+
+function App() {
   return (
     <Router>
       <div>
-        {loading ? (
-          <div className="loader">Loading...</div>
-        ) : (
-          <div>
-
-            <div className="buttons">
-              <button onClick={handleLoginClick}>Login</button>
-              <button onClick={handleRegisterClick}>Register</button>
-            </div>
-
-
-            {showLogin && <Login />}
-            {showRegister && <Register />}
-          </div>
-        )}
+        <nav>
+          <Navbar />
+        </nav>
+        <div>
+          <Routes>
```

Comment:
I think these Routes need to go into their own file. Maybe "shared/routes.js.. 

https://stackoverflow.com/questions/43026690/declaring-react-routes-in-a-separate-file-and-importing

Reply:
Probably also use the component property over element. Like below:
`<Route path="/" exact component={Home} />`

Reply:
Component is used with switch which the latest version of react doesn't have anymore (as far as I can tell). So I had to use Routes and elemenet instead.

Reply:
I'm amending the file structure now to separate routing files